wordle-web/pages/game.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-card style="padding: 5%; margin: 5%">
+    <h-row justify="center" align="center">
+      <h1>ǝlpɹoM </h1>
+      <br />
+      <v-row justify="center" align="center">
+        <v-col
+          style="
+            border: 1px;
+            width: 100px;
+            height: 100px;
+            background-color: grey;
+          "
+        >
+          X
+        </v-col>
+        <v-col
+          style="
+            border: 1px;
+            width: 100px;
+            height: 100px;
+            background-color: grey;
+          "
+        >
+          X
+        </v-col>
+        <v-col
+          style="
+            border: 1px;
+            width: 100px;
+            height: 100px;
+            background-color: grey;
+          "
+        >
+          X
+        </v-col>
+        <v-col
+          style="
+            border: 1px;
+            width: 100px;
+            height: 100px;
+            background-color: grey;
+          "
+        >
+          X
+        </v-col>
+        <v-col
+          style="
+            border: 1px;
+            width: 100px;
+            height: 100px;
+            background-color: grey;
+          "
+        >
+          X
+        </v-col>
```

Comment:
Inline styles like this is usually frowned upon. It would be best to put this style in a css class, that way it is more reusable. However, an even better way to go about this kind of thing is to use vuetify components that support what you're trying to do.